readthedocs/search/api/v3/utils.py

```diff
@@ -0,0 +1,10 @@
+from readthedocs.projects.models import Feature
+
+
+def should_use_advanced_query(projects):
+    # TODO: we should make this a parameter in the API,
+    # we are checking if the first project has this feature for now.
```

Comment:
This is definitely pretty weird/janky. It would be best if we could avoid having this logic entirely, but I don't 100% know the best approach.